components/Card/UserDots.module.scss

```diff
@@ -0,0 +1,84 @@
+.buttonUserDots {
+  display: flex;
+  flex-direction: row;
+  overflow: auto;
+  align-items: center;
+  justify-content: center;
+  background: #ffffff;
+  border-radius: 4px;
+  padding: 0;
+  margin: 0;
+  border: none;
+  flex-shrink: 0;
+}
+
+.buttonUserDots:hover > * {
+  cursor: pointer;
+}
+
+.containerUserDots {
+  position: relative;
+  height: 24px;
+}
+
+.oneUserDot {
+  width: 24px;
+}
+
+.twoUserDots {
+  width: 48px;
+}
+
+.threeUserDots {
+  width: 72px;
+}
+
+.containerUserDots div:nth-child(1) {
+  position: absolute;
+  top: 0px;
+  right: 0px;
+}
+
+.containerUserDots div:nth-child(2) {
+  position: absolute;
+  top: 0x;
```

Comment:
`x` is an invalid css unit.

I prefer that you don't specify unit when assigning `0` as the value.
Change `property: 0px;` to `property: 0;`
![image](https://user-images.githubusercontent.com/3164065/137074168-85799f3f-0d13-4f1a-9b5b-251ecc18842f.png)